src/component/calculator.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './calculate.css';
+import CalcBody from './calcBody';
+
+const calculator = () => (
```

Comment:
- [OPTIONAL] React component name are capitalized to differentiate them from html elements. so i suggest you write this component like below  😄 
```
const Calculator = () => (
  <div className="calculator">
    <CalcBody />
  </div>
);
```